libnetwork/iptables/iptables.go

```diff
@@ -144,8 +144,18 @@ func initCheck() error {
 	return nil
 }
 
-// GetIptable returns an instance of IPTable with specified version
+// GetIptable returns an instance of IPTable with specified version ([IPv4]
+// or [IPv6]). It panics if an invalid [IPVersion] is provided.
 func GetIptable(version IPVersion) *IPTable {
+	switch version {
+	case IPv4, IPv6:
+		// valid version
+	case "":
```

Comment:
Did you try to fix libnetwork/drivers/bridge/link.go on line 78? I think you "just" need to add IPv4 to the `ChainInfo`.

Reply:
I was initially considering that, and to check if I could find code-paths that didn't set an IP-version. There were a lot of places that either accepted or returned a `ChainInfo`, and we must be sure we don't miss some codepath somewhere 😓 

To "some" extend it could also be considered "reasonable" to consider "IPv4" the default (as it always was), so the two above combined, I decided to keep that for a later exercise (but something we should consider looking into)